app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
```

Comment:
- [ ] Database migration is not running successfully on your application, and hence app cannot be run. Kindly ensure to fix this issue in your migration files.

**Migration Error:**
![image](https://user-images.githubusercontent.com/12745474/156239877-2ac36baf-54c4-485d-8850-41a8d3cabb65.png)

Reply:
@rloterh ,

Migration is running perfectly here. Please state the code you are trying to run when you received this error message.

Reply:
@micheaol 
I believe the comment is very clear and to the point. Please, **read the console error log carefully**. 
There is no condition that your app will run the command you have above. Please take your time to do the right thing, as this will reflect in your real job in the nearby future. Cheers!

Reply:
Thank you @rloterh